extensions/amp-story/0.1/amp-story.js

```diff
@@ -372,8 +372,13 @@ export class AmpStory extends AMP.BaseElement {
     const gestures = Gestures.get(this.element,
         /* shouldNotPreventDefault */ true);
 
-    gestures.onGesture(SwipeXYRecognizer, () => {
-      if (this.bookend_.isActive()) {
+    gestures.onGesture(SwipeXYRecognizer, e => {
+      const xSwipe = Math.abs(e.data.deltaX);
+      const ySwipe = Math.abs(e.data.deltaY);
+      const minSwipeDelta = 50;
+      const canShowEducationOverlay =
+          xSwipe > minSwipeDelta || ySwipe > minSwipeDelta;
```

Comment:
This is inaccurate, you should calculate line distance:
```javascript
Math.sqrt(Math.pow(deltaX, 2) + Math.pow(deltaY, 2)) 
  > MIN_SWIPE_FOR_HINT_OVERLAY_PX``

Reply:
oh yeah... makes sense